src/dyn_subgrid/dynFATESLandUseChangeMod.F90

```diff
@@ -0,0 +1,194 @@
+module dynFATESLandUseChangeMod
+
+#include "shr_assert.h"
+
+  !---------------------------------------------------------------------------
+  ! !DESCRIPTION:
+  ! Handle reading of the land use harmonization (LUH2) dataset
+
+  ! !USES:
+  use shr_kind_mod          , only : r8 => shr_kind_r8
+  use shr_log_mod           , only : errMsg => shr_log_errMsg
+  use decompMod             , only : bounds_type, BOUNDS_LEVEL_PROC
+  use abortutils            , only : endrun
+  use dynFileMod            , only : dyn_file_type
+  use dynVarTimeUninterpMod , only : dyn_var_time_uninterp_type
+  use clm_varcon            , only : grlnd
+  use clm_varctl            , only : iulog
+
+  implicit none
+
+  private
+
+  real(r8), allocatable, public :: landuse_transitions(:,:)
```

Comment:
comments about what the local variables are about...

Reply:
addressed via 87b149f86fe87a0e07ef627d64febb304306fd7a